src/main/java/com/oltpbenchmark/DBWorkload.java

```diff
@@ -42,358 +36,79 @@
 import java.io.IOException;
 import java.io.PrintStream;
 import java.sql.SQLException;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 
 public class DBWorkload {
     private static final Logger LOG = LoggerFactory.getLogger(DBWorkload.class);
-
     private static final String SINGLE_LINE = StringUtil.repeat("=", 70);
 
-    private static final String RATE_DISABLED = "disabled";
```

Comment:
@lmwnshn here were the old valued that i converted into the PhaseRateType enum and added the LIMITED value.  see (https://github.com/timveil-cockroach/benchbase/blob/61518dd6ef7ff49ba3c0bf2fec2d26c427184602/src/main/java/com/oltpbenchmark/api/config/PhaseRateType.java#L3)